test/runtests.jl

```diff
@@ -2,6 +2,10 @@ using Test
 using PlayingCards
 using PlayingCards: rank_string
 
+using Random: Xoshiro
+
+rng = Xoshiro(0x0451)
```

Comment:
```suggestion
if VERSION >= v"1.7.0"
    using Random: Xoshiro
    rng = Xoshiro(0x0451)
else
    rng = MersenneTwister()
end
```
Maybe this will work?

Reply:
Should do it, I'll get it in the squashed commits